ts/components/messages/MessagesDeadlines.tsx

```diff
@@ -148,9 +149,9 @@ const generateSections = (
             (accumulator, messageAgendaItem) => {
               // As title of the section we use the ISOString rapresentation
               // of the due_date day.
-              const title = startOfDay(
-                messageAgendaItem.e1.content.due_date
-              ).toISOString();
+              const title = DateFromISOString.encode(
```

Comment:
Why you are encoding here to decode again after?

Reply:
> Why you are encoding here to decode again after?

I kept the same logic already present. Did I miss something?

Reply:
i have restored the method toisoString(), the encoding by DateFromISOStringType is unnecessary